backend/src/v4/models/mitigation.js

```diff
@@ -37,6 +38,36 @@ const fieldTypes = {
 	"associated_activity": "[object String]"
 };
 
+const fieldTypes = {
+	...csvImportFieldTypes,
+	"mitigation_status": "[object String]",
```

Comment:
Mat told me to separare mitigations that are agreed_fully and agreed_partial. IE if there are two identical mitigations where one is agreed fully and the other is partial there should be 2 entries

Reply:
@mosment3DRepo  that doesn't make much sense...

In the perspective of mitigation suggestions, you'll just end up with 2 identical mitigations being suggested

Reply:
When downloading the CSV we would need to understand the mitigation status so that we can understand the mitigation properly.

Is there any way to include the status in the Suggestions table? So that you see two, but understand one is Fully and one is Partially mitigated. 

It is a very very very edge case of course

Reply:
@carmenfan  

Sorry, I didn't mean that the Risk you were using the treatment on would be mitigated, rather the suggestion would indicate whether it has come from a risk that was Fully or Partially mitigated. So that the person applying the mitigation understand its effectiveness.

My solution is more for examples where they are not different, eg I have 3 mitigation suggestions, 2 were Partial and one was Full, therefore I'm more likely to select the one which resulted in a full mitigation (although I will be the one t decide whether the risk I'm applying it to has been fully mitigated). It would also be useful for Admins who may be cleaning up the CSV to understand the mitigations.

Splitting up identical risks into 2 records is a necessary result of that in an edge case, rather than the main goal.

Reply:
If this is what you want I'm still in the opinion that this should be a separate issue - even if we work on it immediately afterwards.

It effectively has nothing to do with the functionality of automatically adding rules to to the risk register - we are adding a new field into the risk register.

If we keep the work separate + having an issue to document what we want to change and how (so we thought through this properly) is useful.

Reply:
@mosment3DRepo

Reply:
after discussing with @carmenfan I am happy to revert these changes for now and consider them in a future issue.

So any Fully or Partially agreed mitigations are treated equally